mungegithub/mungers/release-note-label.go

```diff
@@ -45,7 +45,7 @@ const (
 	releaseNoteActionRequired = "release-note-action-required"
 	releaseNoteExperimental   = "release-note-experimental"
 
-	releaseNoteFormat = `Adding ` + releaseNoteLabelNeeded + ` because the release note process has not been followed.
+	releaseNoteFormat = `Adding %s because the release note process has not been followed.
```

Comment:
nit: s/%s/%q/

Reply:
I don't think this will work -- the comment today looks like:

> Adding do-not-merge/release-note-label-needed because the release note process has not been followed.
One of the following labels is required "release-note", "release-note-action-required", "release-note-experimental" or "release-note-none".
Please see: https://github.com/kubernetes/community/blob/master/contributors/devel/pull-requests.md#write-release-notes-if-needed.

This does not have quotes around the first label name (unlike all the others!)

Reply:
But why wouldn't it work?

Reply:
If we use `%q` here, the resulting comment will have `"label/name"` and this will not match the comment bodies we are trying to clean up.